cfgov/jinja2/v1/_includes/molecules/_vars-content.html

```diff
@@ -0,0 +1,9 @@
+{% macro _privacy_statement() %}
+<p class="short-desc">
```

Comment:
I think it's better to keep it unindented. When you're rendering the macro, including a template, etc _I think_ the indent will be preserved from where it was called.